search_single_spectrum/tools/search_single_spectrum/proteosafe.py

```diff
@@ -0,0 +1 @@
+/home/chrissy/Desktop/github_repos/GNPS_Workflows/CCMS_ProteoSAFe_pythonAPI/ccmsproteosafepythonapi/proteosafe.py
```

Comment:
This will need to be a relative link not an absolute one. Currently points to something on Chrissy's computer